components/org.wso2.carbon.identity.inbound.provisioning.scim2.common/src/main/java/org/wso2/carbon/identity/inbound/provisioning/scim2/common/utils/SCIMClaimResolver.java

```diff
@@ -87,22 +88,23 @@
                     attributes = complexAttribute.getSubAttributesList();
                 }
                 if (attributes != null) {
-                    for (Attribute entry : attributes.values()) {
-                        // if the attribute a simple attribute
+                    attributes.values().stream().forEach(LambdaExceptionUtils.rethrowConsumer(entry -> {
                         if (entry instanceof SimpleAttribute) {
```

Comment:
This code might suite better line-92:99
if (entry instanceof SimpleAttribute &&
                            !entry.getURI().equals(SCIMConstants.CommonSchemaConstants.LOCATION_URI) {
                                setClaimsForSimpleAttribute(entry, claimsMap);
                            }
                        }